providers/network/resources/network.lr

```diff
@@ -236,6 +238,20 @@ pkix.extension {
   value string
 }
 
+// x509 certificate PKIX Subject Alternative Name extension
```

Comment:
```suggestion
// x509 Certificate PKIX Subject Alternative Name Extension
```
for consistency :)

Reply:
Can you please align with @tas50. He changed some titles to be lower case last time https://github.com/mondoohq/cnquery/pull/2372/commits/68231c7062e22c5f207eae04b529ca9344fd1500

Reply:
I'll merge this for now. + 1 to figuring this out, I'm curious to better understand it too. I don't think I'm the right person to cast the decision on this

Reply:
This is my mistake.